dockerfiles/merge/merge.sh

```diff
@@ -21,13 +21,13 @@ else
     ORIGINAL_BRANCH=master
 fi
 
-APACHE_REPO=`git remote show | grep apache || true`
-if [ -z "$APACHE_REPO" ]; then
-    git remote add apache https://github.com/apache/james-project.git
+LINAGORA_REPO=`git remote show | grep linagora || true`
```

Comment:
OUPS?

Reply:
not oups, it's required to merge on linagora instead of apache

Reply:
Why would we do that?

Espacially why would we do that on the build script of an Apache repo? Maybe a level of indirection (env variable) is required?

Also, I don't understand why this would be required for that dependency upgrade...

Reply:
And... We miss the JIRA prefix

Reply:
This change allows to open a PR against a linagora branch, it's what I'm trying to do to here: fix the openpaas 1.5 branch.
There's no apache repository involved here, only linagora repo.
We can add a variable but having a build script that actually build a given branch the right way looks good enough to me.